wallet/package.json

```diff
@@ -4,37 +4,52 @@
   "private": true,
   "main": "./src/index.jsx",
   "dependencies": {
-    "@semantic-ui-react/craco-less": "^1.2.3",
+    "@craco/craco": "^6.4.3",
     "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^12.1.2",
+    "async-mutex": "^0.3.2",
+    "axios": "^0.25.0",
     "bip39": "^3.0.4",
     "bootstrap": "^5.1.3",
     "bootstrap-social": "^5.1.1",
     "crypto-js": "^4.1.1",
+    "ethereumjs-wallet": "^1.0.2",
     "font-awesome": "4.7.0",
+    "general-number": "^1.0.1",
+    "idb": "^7.0.0",
     "js-sha3": "^0.8.0",
-    "nf3": "file:../cli",
+    "keccak": "^3.0.2",
     "prop-types": "^15.7.2",
+    "queue": "^6.0.2",
     "react": "^17.0.2",
+    "react-bootstrap": "^2.1.1",
     "react-dom": "^17.0.2",
+    "react-icons": "^4.3.1",
     "react-redux": "7.2.5",
+    "react-responsive": "^9.0.0-beta.6",
     "react-router-dom": "5.3.0",
     "react-scripts": "4.0.3",
+    "react-to-webcomponent": "^1.5.1",
     "redux": "4.1.1",
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.3.0",
-    "semantic-ui-less": "^2.4.1",
-    "semantic-ui-react": "^2.0.4",
+    "safe-buffer": "^5.2.1",
+    "sass": "^1.49.7",
+    "typescript": "^4.5.5",
+    "wasm-loader": "^1.3.0",
     "web-vitals": "^2.1.0",
-    "web3": "^1.6.0"
+    "web3": "^1.6.0",
+    "zokrates-js": "^1.0.39"
   },
   "scripts": {
-    "start": "REACT_APP_ENVIRONMENT=Localhost craco start",
+    "start": "LOCAL_OPTIMIST=true craco start",
     "start:docker": "REACT_APP_ENVIRONMENT=Docker PORT=3010 craco start",
     "start:ropsten": "REACT_APP_ENVIRONMENT=Ropsten craco start",
-    "build": "REACT_APP_ENVIRONMENT=Ropsten craco build",
-    "deploy": "npm run build && aws s3 sync build/ s3://nightfallv3-wallet --cache-control max-age=172800 --delete && aws configure set preview.cloudfront true && aws cloudfront create-invalidation --distribution-id E9K9C7N0TG7LG --paths \"/*\"",
+    "build": "LOCAL_OPTIMIST=false craco build",
+    "deploy": "npm run build && aws s3 sync build s3://pnf-dev-browser --cache-control max-age=172800 --delete && aws configure set preview.cloudfront true && aws cloudfront create-invalidation --distribution-id ENV3X13MLR5YT --paths \"/*\"",
     "test": "craco test --env=jsdom",
-    "eject": "craco eject"
+    "eject": "craco eject",
+    "e2e-test": "NETWORK_NAME=ganache-nightfall RPC_URL=http://localhost:8546 CHAIN_ID=1337 PRIVATE_KEY=0x4775af73d6dc84a0ae76f8726bda4b9ecf187c377229cb39e1afa7a18236a69e npx synpress run --env TRANSACTIONS_PER_BLOCK=2"
```

Comment:
Instead of hardcoding this, can we make it generic
`process.argv` can be used in nodejs files to access arguments passed to `npm run script-name arg1 arg2`.
`NETWORK_NAME` should be arg1, RPC_URL should be arg2 and so on.

Reply:
Hm I think passing in the `PRIVATE_KEY` in as an argument to the test will be pretty tedious for people. @LijuJoseJJ  do the wallet tests have access to the node `config` file?  If so, can we use the `setupMetamask()` call in synpress and get/add these values from the new `config.TEST_OPTIONS` in `default.js`. Rather than pass the environment variables in the `e2e-test`